src/snipe/api/multisig_reference_QC.py

```
@@ -1,16 +1,59 @@
 import logging
 import warnings
 from typing import Any, Dict, List, Optional
-
+from sklearn.mixture import GaussianMixture
+from scipy.stats import norm, poisson, chisquare
 import numpy as np
 from scipy.optimize import OptimizeWarning, curve_fit
 from snipe.api.snipe_sig import SnipeSig
 from snipe.api.enums import SigType
 import os
 from snipe.api.reference_QC import ReferenceQC
```

Ruff (0.8.2)

`snipe.api.reference_QC.ReferenceQC` imported but unused

Remove unused import: `snipe.api.reference_QC.ReferenceQC`

(F401)